Model/Sales/TrackTraceHolder.php

```diff
@@ -232,6 +234,45 @@ public function validateApiKey($apiKey)
         return $this;
     }
 
+    /**
+     * @param Order\Shipment\Track $magentoTrack
+     *
+     * @param string               $checkoutData
+     *
+     * @return array|mixed|string
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function getLabelDescription(Order\Shipment\Track $magentoTrack, string $checkoutData): string
+    {
+        $labelDescription = $this->helper->getGeneralConfig(
+            'basic_settings/label_description',
+            $magentoTrack->getShipment()->getOrder()->getStoreId()
+        );
+
+        if (! $labelDescription) {
+            return '';
+        }
+
+        $splitLabelDescription = explode(' ', $labelDescription);
+
+        foreach ($splitLabelDescription as $description) {
```

Comment:
Ik zou niet exploden en imploden. Ik zou str_replace gebruiken om alles te vervangen:

$deliveryDate = date('d-m-Y', strtotime($this->convertDeliveryDate($checkoutData)))

$labelDescription = str_replace([self::ORDER_NUMBER, self::DELIVERY_DATE], [$order->getIncrementId(), $deliveryDate], $labelDescription);

Reply:
Done